web/packages/core/package.json

```diff
@@ -41,6 +41,7 @@
         "test": "cross-env TS_NODE_COMPILER_OPTIONS={\\\"module\\\":\\\"commonjs\\\",\\\"verbatimModuleSyntax\\\":false} mocha"
     },
     "dependencies": {
+        "@zip.js/zip.js": "^2.7.6",
```

Comment:
Is there any particular reason for choosing this library? I see [`jszip`](https://www.npmjs.com/package/jszip) is much more popular (6M vs. 50K weekly downloads).

Reply:
Well... No, there's no particular reason. :) `jszip` is indeed much more popular, so I can switch to this library if you want. The development of `@zip.js/zip.js` seems more active though (last publish was a day ago, vs 9 months ago for `jszip`) but I don't know if that matters.

Reply:
I haven't noticed until now but this makes `ruffle.js` (when you compile) grow from ~114kb to ~252kb. Though the file size is still acceptable, this may be a bit too much for such a feature. `jszip` seems smaller in size, so it could be interesting to give it a try.

Reply:
Thanks for checking! My preference is to keep file sizes (both `.js` and `.wasm`) low as possible, because it directly affects download times on Web, so let's stay with `jszip`.